eslint.config.mjs

```diff
@@ -4,6 +4,7 @@ import tseslint from 'typescript-eslint'
 import pluginReact from 'eslint-plugin-react'
```

Comment:
The other of two major changes in this PR. The rest of the changes are results of running `lint --fix`